File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/11364_:::_Parking/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException
	{
		if(test) br=new BufferedReader(new FileReader(new File("in.txt")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine().trim()), menor, mayor, este;

		for(String[] aux; casos--!=0;)
		{
			br.readLine();
			aux=br.readLine().trim().split(" ");

			menor=99;
			mayor=0;
			for (String str : aux)
			{
				este=Integer.parseInt(str);
				if(este>mayor) mayor=este;
				if(este<menor) menor=este;
			}

			System.out.println(2*(mayor-menor));
		}
		//-----------------------------------------------------------
		br.close();
	}
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/01_Super_Easy/11364_:::_Parking/Main.c
#include "stdio.h"

int main()
{
	int casos, mayor, menor, este, tam, i;

	scanf("%d",&casos);
	while(casos--)
	{
		menor=99;
		mayor=0;
		scanf("%d",&tam);
		for(i=0; i<tam; i++)
		{
			scanf("%d",&este);
			if(este>mayor) mayor=este;
			if(este<menor) menor=este;
		}
		printf("%d\n",(2*(mayor-menor)));
	}

	return 0;
}
